content/en/tracing/profiler/enabling.md

```diff
@@ -60,22 +60,29 @@ The Datadog Profiler requires [JDK Flight Recorder][1]. The Datadog Profiler lib
 
 - Datadog strongly recommends that you specify the `service` and `version` as this gives you the ability to slice and dice your profiles across these dimensions. Use environment variables to set the parameters:
 
+## Enabling the allocation profiler
+
+In tracer v0.83.0+, the allocation profiler is disabled by default on Java 15 and lower. To enable it, see [Enabling the allocation profiler][8].
```

Comment:
Nit but I might make this consistent with the troubleshoot page and tweak it because:
1. Generally nice to be consistent between this page and troubleshooting unless there's a good reason to differ.
2. "Overwhelm the profiler" may be a little ambiguous. Will it drop data? Corrupt profiles? Use too much cpu? You and I know the answer ofc but users may not.
3. We miiight want to add something that encourages people to give it a shot in a safe way left everyone just ignore it?

Maybe something like the following in both places.
```suggestion
In dd-java-agent v0.83.0+ and Java 15 and lower, the allocation profiler is opt-in because it can use excessive CPU in allocation-heavy applications. This isn't common, so you may want to try it in a staging environment to see if it affects your application. To enable it, see [Enabling the allocation profiler][8].
```

Feel free to tweak further or skip if you disagree.

Reply:
I like it.